tests/ensure_provider_tests.py

```diff
@@ -114,11 +119,19 @@
 def create_nm_playbook(test_playbook):
     fileroot = os.path.splitext(os.path.basename(test_playbook))[0]
     nm_testfile = fileroot + "_nm.yml"
+    if NM_DEMANDING_TESTS.get(test_playbook, {}) != {}:
+        minimum_nm_version = NM_DEMANDING_TESTS.get(test_playbook, {}).get(
```

Comment:
```suggestion
        minimum_nm_version = NM_DEMANDING_TESTS[test_playbook].get(
```

If you check already if the key is present in the dictionary, there is no need to use `.get()`.

Reply:
Removed the unnecessary `.get()`.